packages/honeycomb-opentelemetry-web/src/span-processor-builder.ts

```diff
@@ -37,14 +37,18 @@ export const configureSpanProcessors = (options?: HoneycombOptions) => {
     honeycombTraceExporters.push(...options.traceExporters);
   }
 
+  // Disable this if a configuration option is present
+  if (options?.disableDefaultTraceExporter !== true) {
+    honeycombTraceExporters.unshift(
+      configureHoneycombHttpJsonTraceExporter(options),
+    );
+  }
+
```

Comment:
We may want a check/logger statement here so that if folks _disable_ the default exporter AND the don't provide any other exporters, there's at least something that's like `console.warn("You gots no exporters configured.")`